masterdata/configuration/programworkflowstates/programWorkflowStates_HIV.csv

```diff
@@ -0,0 +1,10 @@
+Uuid,Void/Retire,Workflow,State concept,Initial,Terminal,_order:1000
+ab3fc1fb-7dd5-463f-b53e-e465db152923,,HIV Program workflow,8ed76778-da7c-4a35-b2c9-fa74fb59a2e0,true,,
+1871ae9f-d9bf-456b-9ea7-93836e97eff0,,HIV Program workflow,56b02820-10a2-4220-9418-dc9a9ed60d86,,true,
+2125a49b-1c83-4a89-b077-c3f3782a666,,HIV Program workflow,1c3363ea-2c60-442f-b9e7-c4ceda50dff2,,,
+58972381-0e85-446c-8842-584c16e72719,,HIV Program workflow,54dbcd5d-6553-4f25-b2d9-f01eee6c9fb7,,,
+491247cf-940f-4efd-aafc-9e132a1ddcd3,,HIV Program workflow,5006AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,
+
```

Comment:
Better to remove additional empty lines, might cause errors while CSV parsing

Reply:
Yes Removed